### DISCUSSION TITLE
Facebook Container and Google Container stopped working with Webapps

### DISCUSSION THREAD

**Discussion by SystemError12** [Floorp-Projects/Floorp#1061] [Q&A]
As the headline says: It seems the Webapps no longer support Containers. 

Few days ago I could still use Youtube and Instagram Webapps, with active Facebook and Google Containers. Now the Webapps will no longer open with those Containers active. 

I don't want to post it as a bug, because it's an issue with the containers primarily. But maybe someone can look into this? The containers stilll work outside of WPA-Web-Apps, but not with WPA. It could be a bug with Floorp or maybe anything with the Containers. 

Anyone else have this?

My Operating System is Windows 11

**Comment by SystemError12** [comment]
Maybe this can be moved to bug section? In my opinion it might actually be a bug. Maybe someone of the floorp team can have a look at it and see what to do about it.

**Comment by CutterKnife** [comment]
issue is here: https://github.com/Floorp-Projects/Floorp/issues/1085

**Comment by SystemError12** [comment]
Hello again. Is there any timeline to bring this feature back? I love the webapps, but would prefer to be able to use them - as was possible before - with containers. In the (now closed) bug post it was said an implementation was planned to bring it back.. .

**Reply by CutterKnife** [reply]
The team is currently refactoring the code for Floorp. The current focus is on the PWA.

**Reply by SystemError12** [reply]
Cool. That sounds great. Thanks so much for making this browser awesome :)
